test/metabase/api/collection_test.clj

```diff
@@ -707,16 +709,17 @@
                  :name     "My Beautiful Collection"
                  :slug     "my_beautiful_collection"
                  :color    "#ABCDEF"
-                 :location "/"})
-               ((mt/user->client :crowberto) :put 200 (str "collection/" (u/get-id collection))
-                {:name "My Beautiful Collection", :color "#ABCDEF"})))))
+                 :location "/"
+                 :parent_id nil})
```

Comment:
Is there a reason `parent_id` went away in the first place? It's present in master